app/src/components/ContributionDetail.vue

```diff
@@ -0,0 +1,104 @@
+            <!-- subgrid -->
+            <div class="grid grid-cols-4 items-center gap-x-4">
+              <!-- img -->
+              <div class="hidden lg:block col-span-1">
+                <!-- do not forget to link the figure to the grant ^^ -->
+                <figure class="aspect-w-16 aspect-h-9 shadow-light cursor-pointer">
+                  <img
+                    class="w-full h-full object-center object-cover group-hover:opacity-90"
+                    :src="contribution.grantLogoURI ? contribution.grantLogoURI : '/placeholder_grant.svg'"
+                  />
+                </figure>
+              </div>
+              <!-- text -->
+              <div class="col-span-4 lg:col-span-3">
+                <a href="#" class="link">{{ contribution.grantName }}</a>
+                <!-- if this tx was via a matching round show round -->
+                <br /><span class="mr-2">via</span>
+                <a href="#" class="link">{{ contribution.roundName }}</a>
+                <!-- end if -->
+              </div>
+            </div>
+          </article>
+
+          <!-- donors + amount -->
+          <article class="col-span-12 md:col-span-6 lg:col-span-4">
+            <!-- subgrid -->
+            <div class="flex justify-between items-center">
+              <!-- donors -->
+              <div>
+                <!-- subgrid -->
+                <div class="flex gap-4 items-center">
+                  <!-- do not forget to link the figure to the donors profile ^^ -->
+                  <figure class="cursor-pointer">
+                    <Jazzicon :address="contribution.address" :key="contribution.address" :width="48" />
+                  </figure>
+                  <div>
+                    <a
+                      :href="getEtherscanUrl(contribution.address, 'address')"
+                      class="link"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      >{{ formatAddress(contribution.address) }}</a
+                    >
+                  </div>
+                </div>
+              </div>
+              <!-- amount -->
+              <div>
+                <div class="text-grey-400">
+                  {{ formatNumber(contribution.amount, 4) }}
+                  {{ contribution.donationToken?.symbol }}
+                </div>
+              </div>
+            </div>
+          </article>
+
+          <!-- transaction time + transaction hash-->
+          <article class="col-span-12 md:col-span-12 lg:col-span-3">
+            <!-- subgrid -->
+            <div class="flex justify-between lg:block lg:text-right">
+              <!-- utc-time -->
+              <div class="text-grey-400">{{ contribution.createdAt }}</div>
+              <!-- tx-hash -->
+              <div class="truncate">
+                <a :href="getEtherscanUrl(contribution.txHash || '', 'tx')" class="link">{{ contribution.txHash }} </a>
```

Comment:
should this open in a new tab / is this fine ?

Reply:
Agreed, should new tab. Just add `target=_blank` attribute to the `a` tag

Reply:
fixed in push!